GUI/QtQuick/qml/WalletProperties/WalletSeed.qml

```diff
@@ -47,16 +47,26 @@ ColumnLayout  {
 
     CustomButton {
         id: confirm_but
-        text: qsTr("Copy seed")
+        text: qsTr("Print PDF")
```

Comment:
According to the implementation PDF is not currently printed, but only saved to a file - will it change?